### DISCUSSION TITLE
How to hide tab text, like icons only?

### DISCUSSION THREAD

**Discussion by rainbowflesh** [mbnuqw/sidebery#1433] [Q&A]
Indent will make icon margin-left, but those button just overflow e.g. sounds and close button, any idea to overflow:hidden or move place to somewhere else?
![QQ截图20240129021512](https://github.com/mbnuqw/sidebery/assets/20819692/a7bf8aa3-7944-4df7-b460-a6818df76677)

---

Here are some sample in TST addone:

![QQ截图20240129021847](https://github.com/mbnuqw/sidebery/assets/20819692/5be7aade-a0ad-4fa1-90a0-42d42fb0b48f)

![QQ截图20240129021836](https://github.com/mbnuqw/sidebery/assets/20819692/f11c5abf-f0e1-4b2b-8f4b-8b29622191af)

**Comment by ghost** [comment]
trying to do the same
